health_cards.gemspec

```diff
@@ -15,14 +15,14 @@ Gem::Specification.new do |spec|
                         'Shaumik Ashraf']
   spec.email         = ['radamson@mitre.org']
   spec.summary       = <<~TEXT
-    Create SMART Health Cards using FHIR and Verifiable Credentials for secure and decentralized#{' '}
+    Create SMART Health Cards using FHIR and Verifiable Credentials for secure and decentralized 
     presentation of clinical information.
   TEXT
   spec.description = <<~TEXT
-    Health Cards implements SMART Health Cards, a secure and decentralized framework that allows#{' '}
-    people to prove their vaccination status or medical test results. It is built on top of FHIR 4#{' '}
-    healthcare interoperability standards and W3C Verifiable Credentials. It allows conversion of
-    medical data into JWS which may then be embedded into QR codes.
+    Health Cards implements SMART Health Cards, a secure and decentralized framework that allows 
+    people to prove their vaccination status or medical test results. It is built on top of FHIR R4 
+    health data exchange standards and W3C Verifiable Credentials. It allows conversion of
+    clinical data into JWS which may then be embedded into QR codes.
```

Comment:
It looks like this is very similar, but slightly different to the README.md descriptions. Is there a reason the last sentence about the other 2 methods of exporting a health card was not updated here in the .gemspec?

Reply:
that was my bad